postcodeIO.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, request, jsonify
+
+app = Flask(__name__) 
+
+@app.route('/getHouses', methods=['POST'])
+def postcodesIO():
+    frontData = request.get_json()
+    latitude = frontData['lat']
+    longitude = frontData['longlngitude']
+    radius = frontData['radius']
+
+    #Default limit? just in case the user does not stablish it
+    houseLimit = frontData['limit']
+
+    listOfPostcodes = callAPI(latitude,longitude,radius)
+    listOfPostcodes[houseLimit:]
+    # houseList = callKayleshmodule
```

Comment:
Do you know what Kaylesh's module will require? (the method signatures)

If so you can implement your part here with a black box strategy and if you need to test, you can mock it..

Reply:
I have decided to write my code into David's program instead of creating another class. This script will now instantiate an SQLQueue class directly.